package.json

```diff
@@ -36,18 +36,14 @@
   "devDependencies": {
     "@babel/core": "7.5.5",
     "@wordpress/babel-preset-default": "4.4.0",
-    "@wordpress/browserslist-config": "2.6.0",
-    "babel-plugin-transform-async-generator-functions": "6.24.1",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-plugin-transform-react-jsx": "6.24.1",
-    "babel-plugin-transform-runtime": "6.23.0",
-    "babel-preset-env": "1.7.0",
     "@wordpress/blocks": "6.5.0",
+    "@wordpress/browserslist-config": "2.6.0",
     "@wordpress/components": "8.1.0",
     "@wordpress/date": "3.4.0",
     "@wordpress/dependency-extraction-webpack-plugin": "1.1.0",
     "@wordpress/editor": "9.5.0",
     "@wordpress/element": "2.6.0",
+    "@wordpress/eslint-plugin": "^2.4.0",
```

Comment:
I'm not sure this is explicitly needed here because we have access by virtue of it being a dependency of `@wordpress/scripts` (I could be wrong though)?

Reply:
You're right, `@wordpress/scripts` includes `@wordpress/eslint-plugin` so it's available even if we don't declare it in `package.json`. But we are calling `@wordpress/eslint-plugin` directly from `.eslintrc.js`, so our project directly depends on it.

My concern is that if a future version of `@wordpress/scripts` didn't include `@wordpress/eslint-plugin`, that would break our project. That's why I added it to `package.json`. Does that make sense or you think we shouldn't define it anyway?

Reply:
For a devDependency I'm not worried about breakage.  If it got removed from `@wordpress/scripts` it'd be easy enough to add the dependency then (errors would show up in our IDE's).  In my mind it's preferable to keep our package.json as lean as possible from a standpoint of maintainability and offloading that to `@wordpress/scripts` is beneficial for that reason.

Reply:
Fair enough. I removed it in a6dff045001238daf3b8122edfdcbc29db66bf1b.